tests/test_price_parsing.py

```diff
@@ -1944,6 +1946,21 @@ def __eq__(self, other):
     Example(None, '15.08.2017',
             None, None, None),
 
+    Example(None, '0€ until May, 2005, 35€ afterwards',
+            '€', '0', 0),
+
+    Example(None, '2019-08-19: 22 USD',
+            'USD', '22', 22),
+
+    Example(None, '2105 EUR at July, 2004',
+            'EUR', '2105', 2105),
+
+    Example(None, '$10 EUR during March, 2016',
+            '$', '10', 10),
+
+    Example(None, '$10 EUR at March, 2016 or 2019-08-19',
+            '$', '10', 10),
```

Comment:
hey @bulatbulat48!  All these tests are added to PRICE_PARSING_EXAMPLES_XFAIL, i.e. a list of examples which are known _not_ to work. Could you please move them to PRICE_PARSING_EXAMPLES_BUGS_CAUGHT?

Reply:
Got it. Moved tests. Thank you for review!